tests/simulator/test_init_from_restart.py

```
@@ -0,0 +1,84 @@
+import sys
+import copy
+import unittest
+import subprocess
```

Ruff

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)